samples/GuestConfiguration/package-samples/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Policy Guest Configuration samples
+
+These samples are shared to help authors
+to create their own configurations
+and resource modules
+for us in
+[custom Guest Configuration](https://docs.microsoft.com/en-us/azure/governance/policy/how-to/guest-configuration-create) projects.
```

Comment:
Even with the link to the docs it may be helpful to have an example of how to consume one specific config/module from the list in this repo.

Reply:
I like this idea.  Give me a day or two and I will follow up.

Reply:
Added .example-project folder including script to demonstrate how a package is created.